src/AppBundle/Controller/UserController.php

```diff
@@ -73,4 +74,21 @@ public function avatarAction($username, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/register/resend-email", name="register_resend_email", methods={"POST"})
+     * @Template()
+     */
+    public function resendRegistrationEmailAction(Request $request, UserManagerInterface $userManager, MailerInterface $mailer)
+    {
+        if($request->request->get("email") !== NULL) {
```

Comment:
You should assign `$request->request->get("email")` into a variable (like below), and use this variable instead of calling the function twice